### PR TITLE
Combine various webhook content pages

### PR DESCRIPTION
**Pull request by neflyte**
#### Summary

Combined various pages describing both incoming and outgoing webhooks into one page per topic. The previous versions of the webhook pages were moved into the `archive` directory.

### CODE REVIEW

File: site/content/integrate/apps/_index.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Read the [JavaScript start guide]({{< ref "quickstart/quick-start-js" >}}) or [G
...
+The built-in [incoming webhook]({{< ref "/integrate/webhooks/incoming" >}}) is great for simple use cases. It requires the incoming payload to contain a valid {{< newtabref href="https://pkg.go.dev/github.com/nhannv/mattermost-server/model#Post" title="Post" >}} JSON object - but some external systems don't allow the customization of the fields included in the webhook payload. Compared to built-in webhooks, an App or plugin can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
```
Comment (cwarnermm) [inline site/content/integrate/apps/_index.md:44]:
```suggestion
The built-in [incoming webhook]({{< ref "/integrate/webhooks/incoming" >}}) is great for simple use cases. It requires the incoming payload to contain a valid {{< newtabref href="https://pkg.go.dev/github.com/nhannv/mattermost-server/model#Post" title="Post" >}} JSON object, but some external systems don't allow customization of the fields included in the webhook payload. Compared to built-in webhooks, an App or plugin can be customized to receive an HTTP webhook posting from another system, and can process the incoming data then augment it or make an actionable message in a channel.
```

File: site/content/integrate/webhooks/_index.md [hunk 2]
```diff
@@ -6,4 +6,23 @@ aliases:
   - /integrate/webhooks/golang-webhook/
 ---
 
-Webhooks provide a simple way to post messages to a channel ([incoming webhooks]({{< ref "incoming" >}})) and trigger external actions ([outgoing webhooks]({{< ref "outgoing" >}})).
+Mattermost supports webhooks to easily integrate external applications into the server.
+
+### Incoming webhooks
+
+Use incoming webhooks to post messages to Mattermost Public Channels, Private Channels, and Direct Messages. Messages are sent via an HTTP POST request to a Mattermost URL generated for each application and contain a specifically formatted JSON payload in the request body.
```
Comment (cwarnermm) [inline site/content/integrate/webhooks/_index.md:13]:
```suggestion
Use incoming webhooks to post messages to Mattermost public channels, private channels, and direct messages. Messages are sent via an HTTP POST request to a Mattermost URL generated for each application and contain a specifically formatted JSON payload in the request body.
```

File: site/content/integrate/webhooks/incoming/_index.md [hunk 3]
```diff
@@ -2,5 +2,193 @@
...
+  - /integrate/webhooks/incoming/using-incoming-webhooks/
 ---
-Incoming webhooks let you POST some data to a Mattermost endpoint to create a message in a channel.
+## Create an incoming webhook
+
+Let's learn how to create a simple incoming webhook that posts the following message to Mattermost.
+
+![image](incoming_webhooks_create_simple.png)
+
+1. First, go to **Product menu > Integrations > Incoming Webhook**.
```
Comment (cwarnermm) [inline site/content/integrate/webhooks/incoming/_index.md:15]:
```suggestion
1. In Mattermost, go to **Product menu > Integrations > Incoming Webhook**.
```

File: site/content/integrate/webhooks/incoming/_index.md [hunk 4]
```diff
@@ -2,5 +2,193 @@
...
+    - If you don't have the **Integrations** option in your Main Menu, incoming webhooks may not be enabled on your Mattermost server or may be disabled for non-admins. They can be enabled by a System Admin from **System Console > Integrations > Integration Management**. Then continue with the steps below.
```
Comment (cwarnermm) [inline site/content/integrate/webhooks/incoming/_index.md:16]:
```suggestion
    - If you don't have the **Integrations** option, incoming webhooks may not be enabled on your Mattermost server or may be disabled for non-admins. They can be enabled by a System Admin from **System Console > Integrations > Integration Management**. Once incoming webhooks are enabled, continue with the steps below.
```

File: site/content/integrate/webhooks/incoming/_index.md [hunk 5]
```diff
@@ -2,5 +2,193 @@
...
+2. Select **Add Incoming Webhook** and add name and description for the webhook. The description can be up to 500 characters.
```
Comment (cwarnermm) [inline site/content/integrate/webhooks/incoming/_index.md:17]:
```suggestion
2. Select **Add Incoming Webhook** and a add name and description for the webhook. The description can be up to 500 characters.
```

File: site/content/integrate/webhooks/incoming/_index.md [hunk 6]
```diff
@@ -2,5 +2,193 @@
...
+Content-Type: text/plain
+X-Request-Id: hoan6o9ws7rp5xj7wu9rmysrte
+X-Version-Id: 4.7.1.dev.12799cd77e172e8a2eba0f9091ec1471.false
+Date: Sun, 04 Mar 2018 17:19:09 GMT
+Content-Length: 2
+
+ok
+```
+
+All webhook posts will display a `BOT` indicator next to the username in Mattermost clients, to help prevent against [phishing attacks](https://en.wikipedia.org/wiki/Phishing).
```
Comment (cwarnermm) [inline site/content/integrate/webhooks/incoming/_index.md:66]:
```suggestion
All webhook posts will display a `BOT` indicator next to the username in Mattermost clients to help prevent against [phishing attacks](https://en.wikipedia.org/wiki/Phishing).
```

**Review by cwarnermm — COMMENTED** [review]
Thanks, @neflyte. I've provided editorial feedback inline.

File: site/content/integrate/webhooks/incoming/_index.md (hunk 5 above)
Comment (neflyte) [reply site/content/integrate/webhooks/incoming/_index.md:17]: It looks like the `a` should come after the `add`.

Comment (cwarnermm) [reply site/content/integrate/webhooks/incoming/_index.md:17]: Thanks! Suggestion updated!

**Review by cwarnermm — APPROVED** [review]

**Review by azigler — APPROVED** [review]
Looks good, thanks @neflyte and @cwarnermm! 🎸 🛡️

**Pull request merged**